pkg/channel/fanout/fanout_event_handler.go

```diff
@@ -100,18 +100,19 @@ func NewFanoutEventHandler(
 	config Config,
 	reporter channel.StatsReporter,
 	eventTypeHandler *eventtype.EventTypeAutoHandler,
-	channelAddressable *duckv1.KReference,
+	channelRef *duckv1.KReference,
 	channelUID *types.UID,
 	eventDispatcher *kncloudevents.Dispatcher,
 	receiverOpts ...channel.EventReceiverOptions,
+
 ) (*FanoutEventHandler, error) {
 	handler := &FanoutEventHandler{
 		logger:             logger,
 		timeout:            defaultTimeout,
 		reporter:           reporter,
 		asyncHandler:       config.AsyncHandler,
 		eventTypeHandler:   eventTypeHandler,
-		channelAddressable: channelAddressable,
+		channelAddressable: channelRef,
```

Comment:
```suggestion
		channelRef: channelRef,
```
I think this is clearer - when I looked at this initially I thought this was a mistake because we were passing a `KReference` to what I though wanted a `Adressable` type